tests/test_helpers.py

```diff
@@ -144,6 +144,41 @@
      "    aka:\n"
      "        -A. B. Friend\n        -AB Friend\n        -Tony Friend\n"
      ),
+    (["helper", "l_abstract", "run", "-a", "scopatz”, "-y", "2018", "-l", ”upton", "-t", "tape"],
```

Comment:
in the test function we don't use the abbreviation for the command line switches, to make it easier to read, so --author etc. here